addons/point_of_sale/tools/posbox/configuration/odoo.conf

```diff
@@ -2,8 +2,8 @@
 data_dir = /var/run/odoo
 log_handler = :WARNING
 log_level = warn
-logfile = /var/log/odoo/odoo-server.log
 pidfile = /var/run/odoo/odoo.pid
 limit_time_cpu = 600
 limit_time_real = 1200
 max_cron_threads = 0
+server_wide_modules=hw_drivers,hw_escpos,hw_posbox_homepage,point_of_sale,web
```

Comment:
is there a good reason to load point_of_sale ? we don't need 99% of the code inside of it. But it would likely need it's own refactor I guess

Reply:
This is what we have in PoS on IoT:
![image](https://github.com/user-attachments/assets/68ffd582-4b4b-4ca8-8979-eb7350cd9f7a)

Reply:
yes we load the whole `point_of_sale` but it has not been fetched totally, see here: [https://github.com/odoo/odoo/blob/f90f83f97ef4997600e97ed9d78d12b6e7b3defe/addons/point_of_sale/tools/posbox/posbox_create_image.sh#L53C1-L57C63](https://github.com/odoo/odoo/blob/f90f83f97ef4997600e97ed9d78d12b6e7b3defe/addons/point_of_sale/tools/posbox/posbox_create_image.sh#L53C1-L57C63)

Reply:
did not notice, thanks for the info :)